client/css/style.css

```diff
@@ -573,6 +573,7 @@ button {
 }
 #chat .wrap,
 #chat .text a {
+	cursor: pointer;
```

Comment:
MIght be that one of us is missing something. My point is, when I create a link like so: `<a data-chan="#channel">` it will not have the pointer cursor due to the missing `href`. At least firefox behaves like that. To solve that I added `cursor: pointer;`.